src/ensembl/production/metadata/admin/models.py

```diff
@@ -19,13 +19,15 @@ class Assembly(models.Model):
     assembly_id = models.AutoField(primary_key=True)
     ucsc_name = models.CharField(max_length=16, blank=True, null=True)
     accession = models.CharField(unique=True, max_length=16)
+    alt_accession = models.CharField(unique=True, max_length=16, null=True)
```

Comment:
This one can't be unique, because we might have multiple GCF for the same original GCA